airflow/providers/amazon/aws/links/emr.py

```diff
@@ -0,0 +1,30 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.providers.amazon.aws.links.base_aws import BASE_AWS_CONSOLE_LINK, BaseAwsLink
+
+EMR_CLUSTER_LINK = (
```

Comment:
Small nit: Any reason not to define this inline inside the class for `format_str` (it's only used once there). It seems private to the `EmrClusterLink` class and it will get messy if more link classes are added to this module in the future (with all the format strings defined outside the classes).

Reply:
Initially I grab main idea from Google Cloud Provider

https://github.com/apache/airflow/blob/047a6162b0b4cbf07fe2fd978e335839a7d3900b/airflow/providers/google/cloud/links/bigquery.py#L26-L35

So, actually there is no reason set as constant, and I could directly set to `format_str`